cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -23,6 +23,11 @@
 
 
 class Robot(BaseSalesforceTask):
+    task_docs = """Runs Robot test cases using a browser, if
```

Comment:
Needs consistent indenting in order to format correctly.

```suggestion
    task_docs = """
    Runs Robot test cases using a browser, if
```